scripts/seed_db.py

```diff
@@ -0,0 +1,160 @@
+def number_to_table(number):
+    letter = chr((int(number) - 1)//15 + 65)
+    num = int(number) - (int(number) - 1)//15 * 15
+    return str(letter) + str(num)
+
+
+# checks if hacker responds with a table number, so they can't move
+def needs_to_stay(response):
+    return re.search('([a-zA-Z]+\d+)', response)
+
+
+# best domain name......
+def format_challenges(attempted_challenges):
+    if attempted_challenges is "":
+        return ""
+    output = []
+    challenges = attempted_challenges.split(',')
+    for challenge in challenges:
+        data = challenge.split(' - ')
+        prize = {
+            'company': data[1],
+            'challenge': data[0],
+            'winner': 'false'
+        }
+        output.append(prize)
+        json_output = json.dumps(output)
+    return json_output
+
+
+def already_in_db():
+    r = requests.get("http://127.0.0.1:5000/api/projects")
+    data = re.findall('\"project_name\": \"(.+)\"', r.text)
+    projects = list(map(lambda x: x.strip(), data))
+    return projects
+
+
+# parses devpost csv and separates hackers into two groups
+# can't move: assigns table and spot, adds notMoving list
+# can move: adds to moving list
+def parse_CSV(reader):
+    already_stored = already_in_db()
+    for row in reader:
+        project_name = row["Submission Title"]
+        project_url = row["Submission Url"]
+        attempted_challenges = format_challenges(row["Desired Prizes"])
+        #attempted_challenges = row["Desired Prizes"]
+        response = row[gdi_devpost]
+
+        name = row['Submission Title'].strip()
+        if name not in already_stored:
+            staying = needs_to_stay(response)
+            if staying is not None:
+                notMoving[project_name] = Project(project_url, attempted_challenges)
+                assignments[table_to_number(staying.group(0))] = name + " | "
+                notMoving[project_name].table_number = staying.group(0)
+            elif "No" not in response and \
+                 "no" not in response and \
+                 "NO" not in response and \
+                 response is not "":
+                print("Manually handle " + project_name)
+                print("Response: " + response)
+            else:
+                moving[project_name] = Project(project_url, attempted_challenges)
+
+
+# fancy way of seeding
+def fancy_seed_hackers():
+    place = 0
+    skip = 391//len(moving)
+    for hacker in moving:
+        while (assignments[place] != "None | "):
+            place = (place + 1) % 391
+        assignments[place] = hacker + " | "
+        moving[hacker].table_number = number_to_table(place)
+        place = (place + skip) % 391
+
+
+# normal seeding
+def seed_hackers():
+    place = 0
+    for hacker in moving:
+        while (assignments[place] != "None | "):
+            place = place + 1
+        assignments[place] = hacker + " | "
+        moving[hacker].table_number = number_to_table(place)
+
+
+def add_project(projects):
```

Comment:
This is okay for the script - TODO is to insert the project into the DB directly using pymongo (this should be done in server.py) instead of using the request module.

[Bulk inserts](http://api.mongodb.com/python/current/tutorial.html#bulk-inserts) can also be done using pymongo if we first construct an array of objects (probably more efficient than doing 200+ insert requests to the DB)